docs/en/operations/system-tables/processors_profile_log.md

```diff
@@ -0,0 +1,73 @@
+# system.processors_profile_log {#system-processors_profile_log}
+
+This table contains profiling on processors level (that you can find in [`EXPLAIN PIPELINE`](../../sql-reference/statements/explain.md#explain-pipeline)).
+
+Columns:
+
+-   `event_date` ([Date](../../sql-reference/data-types/date.md)) — The date when the event happened.
+-   `event_time` ([DateTime64](../../sql-reference/data-types/datetime64.md)) — The date and time when the event happened.
+-   `query_id` ([String](../../sql-reference/data-types/string.md)) — ID of the query
+-   `name` ([String](../../sql-reference/data-types/string.md)) — Name of the processor.
```

Comment:
Well, many processors may have similar names. I suppose, we add a separate event for each processor, still it's hard to understand which one is responsible for what.

Don't know what should be a better solution. Maybe, enumerate processors for a query, and also add info about the graph structure. Like, parents (or children) array.

Reply:
Yeah, name is too short to identify.
Although graph structure sounds complicated for these table.
Maybe description will be enough?
I will try description first and if it will not be enough, then I will add something more.

Reply:
For each processor we can just store an array of parents. I think it won't be much heavier.
It would be possible to build a graph with a query from this table.

Reply:
Done.